godotproject/Menus/InGameMenu/SwitchMenuButton.gd

```diff
@@ -1,9 +1,9 @@
 extends Button
 
 export (NodePath) var menu_control_path
-onready var menu_control = get_node(menu_control_path)
+onready var menu_control = get_node(menu_control_path) if menu_control_path else null
```

Comment:
what happens when you try and use the menu in the case where this is set to null?

Reply:
An error would occur. The game could freeze/crash, or maybe it would keep running with the button doing nothing. The only case where that would happen is if we forgot to set the Menu Control Path in the editor, in which case we would presumably catch it when we went to test the button and found it didn't work.

Reply:
ok

Reply:
I just tested it, it causes the game to stop and break into the debugger when the button is clicked. I'm putting in a quick change to print out an error